CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 - [task] `TaskDefinition.properties.required` is now optional to align with the specification [#10015](https://github.com/eclipse-theia/theia/pull/10015)
 - [core] `setTopPanelVisibily` renamed to `setTopPanelVisibility` [#10020](https://github.com/eclipse-theia/theia/pull/10020)
 - [application-manager] break `rebuild` API: second argument is now an optional object instead of an optional array.
+- [output] moved `output-channel` from `common` to `browser` [#10154](https://github.com/eclipse-theia/theia/pull/10154)
+- [output] moved `output-preferences` from `common` to `core` [#10154](https://github.com/eclipse-theia/theia/pull/10154)
```

Comment:
```suggestion
- [output] moved `output-preferences` from `common` to `browser` [#10154](https://github.com/eclipse-theia/theia/pull/10154)
```

I think?

Reply:
Thanks! I'll make the update and move the entries to `1.19.0` :+1: